e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -1990,6 +1990,265 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
+        "Product → Vendor",
+        "Product → Price",
+        "Product → Rating",
+        "Product → Created At",
+        // 1st stage - Aggregations & breakouts
+        "Created At: Month",
+        "Category",
+        "Created At: Year",
+        "Count",
+        "Sum of Total",
+        // 2nd stage - Custom columns
+        "5 * Count",
+        // 2nd stage - Reviews #2 (explicit join)
+        "Reviews - Created At: Month → ID",
+        "Reviews - Created At: Month → Product ID",
+        "Reviews - Created At: Month → Reviewer",
+        "Reviews - Created At: Month → Rating",
+        "Reviews - Created At: Month → Body",
+        "Reviews - Created At: Month → Created At",
+        // 2nd stage - Aggregations & breakouts
+        "Category",
+        "Created At",
+        "Count",
+        "Sum of Rating",
+      ]);
+
+      // 1st stage - Orders
+      getClickMapping("ID").click();
+      popover().findByText("ID").click();
+
+      // 1st stage - Custom columns
+      getClickMapping("Net").click();
+      popover().findByText("User → Longitude").click();
+
+      // 1st stage - Reviews #1 (explicit join)
+      getClickMapping("Reviews - Product → Reviewer").click();
+      popover().findByText("Product → Category").click();
+
+      // 1st stage - Products (implicit join with Orders)
+      getClickMapping("Product → Title").first().click();
+      popover().findByText("Product → Category").click();
+
+      // 1st stage - People (implicit join with Orders)
+      getClickMapping("User → Longitude").click();
+      popover().findByText("User → Longitude").click();
+
+      // 1st stage - Products (implicit join with Reviews)
+      getClickMapping("Product → Vendor").last().click();
+      popover().findByText("Product → Category").click();
+
+      // 1st stage - Aggregations & breakouts
+      getClickMapping("Category").first().click();
+      popover().findByText("Product → Category").click();
+
+      // 2nd stage - Custom columns
+      getClickMapping("5 * Count").click();
+      popover().findByText("Count").click();
+
+      // 2nd stage - Reviews #2 (explicit join)
+      getClickMapping("Reviews - Created At: Month → Rating").click();
+      popover().findByText("ID").click();
+
+      // 2nd stage - Aggregations & breakouts
+      getClickMapping("Count").last().click();
+      popover().findByText("User → Longitude").click();
+
+      customizeLinkText(`Created at: {{${CREATED_AT_COLUMN_ID}}} - {{count}}`);
+
+      cy.get("aside").button("Done").click();
+      saveDashboard({ waitMs: 250 });
+
+      getDashboardCard()
+        .findAllByText("Created at: May 2022 - 1")
+        .first()
+        .click();
+
+      cy.wait("@dataset");
+
+      cy.location("pathname").should("equal", "/question");
+      cy.findByTestId("app-bar").should(
+        "contain.text",
+        `Started from ${targetQuestion.name}`,
+      );
+
+      // TODO: https://github.com/metabase/metabase/issues/46774
```

Comment:
I left [a comment](https://github.com/metabase/metabase/issues/46774#issuecomment-2437234105) in the issue to clean this up later.